Jenkinsfile

```diff
@@ -106,14 +106,17 @@ void popArtifactFile(String FILE_NAME) {
 TestsReport = '| Test name | Status |\r\n| ------------- | ------------- |'
 testsReportMap  = [:]
 testsResultsMap = [:]
+TestsReportXML = '<testsuite name=\\"PSMDB\\">\n'
```

Comment:
Is this first step to adding duration to test report or you have some other reason for adding this?

Reply:
Yep) You're right. I decided to split these cases in several PR.